.gitignore

```diff
@@ -1,9 +1,10 @@
 .idea
-*/target
```

Comment:
I would leave also */target, why did you remove that?

Reply:
`*/**/target` also covers `*/target`

Reply:
Sure, you're right 👍